chain/network/src/peer/peer_actor.rs

```diff
@@ -210,42 +241,46 @@ impl PeerActor {
             addr: network_state.config.node_addr.clone(),
             account_id: network_state.config.validator.as_ref().map(|v| v.account_id()),
         };
+        let (send, recv) = tokio::sync::oneshot::channel();
```

Comment:
comment on what is this channel used for.

Reply:
done